projects/homework_103_nourish/src/main.c

```diff
@@ -0,0 +1,53 @@
+#include "adc.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+static void interrupt_callback_fn(const GpioAddress *address, void *context) {
+  GpioAddress *address = (GpioAddress *)context;
+  uint16_t value = 0;
+  adc_read_converted(&address, *value);
+  LOG_DEBUG("Converted ADC value is %d\n", value);
+}
+int main(void) {
```

Comment:
Nitpick: there should be a blank line between functions.

Reply:
done